### DISCUSSION TITLE
GIS users Semi-Automatic Classification Plugin

### DISCUSSION THREAD

**Discussion by Marius-ops** [semiautomaticgit/SemiAutomaticClassificationPlugin#213] [General]
Dear All;
I am a Qgis user for satellite image processing.
I am writing  you because I noticed that in the preprocessing of Landsat 7 data, the Semi-Automatic Classification Plugin does not convert the values ​​of the counts correctly, both with respect to the manual calculation with the formula, and with respect to the previous release version of the plugin (7.8 .27) which instead shows exactly the converted value of the count and also verified with the formula.
The version I have installed of the Semi-Automatic Classification Plugin is 7.10.5. and the Qgis version 3.22
I used the:
Band 2 Landsat 7 ETM +; LE07_L1TP_188032_20131205_20200907_02_T1_B2;
The MTL file. txt; LE07_L1TP_188032_20131205_20200907_02_T1_MTL;
The formula is: (π * (Brescale+Grescale*Band_value) *(Earth_Sun_distance^2 ))/ ( Solar_irradiance * sin⁡ (Sun_elevation*π/180));

Is it possible to know if I am doing something wrong in the preprocessing operations?
At least it is possible to have the previous plugin (7.8.27), which gives me the right values?

thank you so much

Mariano

**Comment by semiautomaticgit** [comment]
Hello @Marius-ops ,
thank you for reporting this issue. I have fixed this in the updated version 7.10.6.
Please download this new version.

Cheers

**Reply by Marius-ops** [reply]
Hiiiii,
thank you so much, it works now

Mariano
